sdk/translation/ai-translation-text-rest/samples-dev/breakSentence.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary This sample demonstrates how to make a simple call to the Azure Text Translator service to get sentences' boundaries.
+ */
+import TextTranslationClient, {
+  ErrorResponseOutput,
+  TranslatorCredential,
+  InputTextItem,
+  FindSentenceBoundariesQueryParamProperties,
+  BreakSentenceItemOutput,
+  isUnexpected,
+} from "@azure-rest/ai-translation-text";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const endpoint = process.env["ENDPOINT"] || "https://api.cognitive.microsofttranslator.com";
+const apiKey = process.env["TEXT_TRANSLATOR_API_KEY"] || "<api key>";
+const region = process.env["TEXT_TRANSLATOR_REGION"] || "<region>";
+
+export async function main() {
+  console.log("== Get Sentence Boundaries sample ==");
+
+  const translateCedential = new TranslatorCredential(apiKey, region);
+  const translationClient = TextTranslationClient(endpoint, translateCedential, undefined);
+
+  const inputText: InputTextItem[] = [{ text: "zhè shì gè cè shì。" }];
+  const parameters: FindSentenceBoundariesQueryParamProperties & Record<string, unknown> = {
+    language: "zh-Hans",
+    script: "Latn",
+  };
+  const breakSentenceResponse = await translationClient.path("/breaksentence").post({
+    body: inputText,
+    queryParameters: parameters,
+  });
+
+  if (breakSentenceResponse.status !== "200") {
+    const error = breakSentenceResponse.body as ErrorResponseOutput;
+    throw error.error;
+  }
+
+  if (isUnexpected(breakSentenceResponse)) {
+    throw breakSentenceResponse.body;
+  }
+
+  const breakSentences = breakSentenceResponse.body as BreakSentenceItemOutput[];
```

Comment:
```suggestion
  const breakSentences = breakSentenceResponse.body;
```

The explicit cast is not needed here, checking isUnexpected in line 44 will take care of narrowing down the type so the response will be this:

![image](https://user-images.githubusercontent.com/21109913/230661433-77b620d1-2307-444d-a3c9-b0dfef907db5.png)

Reply:
done